docker-multinode/test/util.sh

```diff
@@ -43,14 +43,16 @@ clone_k8s() {
   K8S_VERSION=${K8S_VERSION:-${LATEST_STABLE_K8S_VERSION}}
   git clone https://github.com/kubernetes/kubernetes.git
   cd ${K8S_LOCATION_PATH}
-  git checkout -b ${K8S_VERSION}
```

Comment:
`git checkout -b ${K8S_VERSION}` just creates local branch `${K8S_VERSION}` from HEAD. 

It should be:
`git checkout ${K8S_VERSION}`
or 
`git checkout -b ${K8S_VERSION} ${K8S_VERSION}`